src/ros_metamodels/rossystem_metamodel_core.py

```diff
@@ -74,7 +74,70 @@ def __init__(self, name, reference, namespace=""):
 
     def dump_xtext_model(self, indent, name_type, interface_type):
         return ("%s%s { name '%s' %s '%s'}") % (
-            indent, name_type, self.resolved, interface_type, self.itype.replace("/", "."))
+            indent, name_type, self.resolved, interface_type, self.reference.replace("/", "."))
+
+class RosParameter(object):
```

Comment:
The RosParameter correspond to the following model:

![image](https://user-images.githubusercontent.com/650568/115840619-29aa5380-a41c-11eb-897a-74481f003320.png)

where "parameter" is a reference to a "parameter" (i.e. https://github.com/ipa320/ros_model_parser/blob/dbf423844f25ef25118c85dd0b6fc8e1f78dfc1c/src/ros_metamodels/ros_metamodel_core.py#L132)

In Xtext this is:

```
'RosParameter'
(name=EString)?
'{'
	('ns' ns=EString)?
	'RefParameter' parameter=[ros::Parameter|EString]
	('value' value=ParameterValue)?
'}';
```

This means the attributes here should be:

```
self.name
self.namespace
self.reference
self.value
```

We can also use exactly the same name as the ones used by the Xtext grammar (name, ns, RefParameter and value).

The reference should be defined in the same way than for the `RosInterface` class, this should be "package_name.artifact_name.node_name.parameter_name"

For the ROS model I added the additional attribute 'fullname' https://github.com/ipa320/ros_model_parser/blob/dbf423844f25ef25118c85dd0b6fc8e1f78dfc1c/src/ros_metamodels/ros_metamodel_core.py#L134-L137, at the end it is the composition of the namespace+name, but I think it can be useful to have this notation at model level.